### PR TITLE
Fetch `schemas` in mssql and pgsql in alphabetical order

### LINKED ISSUE #1176: sort schema alphabetically
The schema list in Postgres connections is not sorted making it very difficult to use when there are a large number of schemes.

I can't post a screenshot due to the nature of the schema names, etc.

### PR DESCRIPTION
**Pull request by bombillazo**
Return the fetched schemas in alphabetical order, resolved #1176

----

Thank you for your contribution! 
Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings
- [x] You have made the needed changes to the docs
- [x] You have written a description of what is the purpose of this pull request above

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Place your settings in this file to overwrite default and user settings.
 {
   "eslint.enable": true,
+  "deno.enable": false,
```
Comment (gjsjohnmurray) [inline .vscode/settings.json]: Please remove this added line.

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (bombillazo) [reply .vscode/settings.json]: Is SQLTools using Deno? For those with the Deno VScode extensions, development is hampered by the Deno extension thinking this is a Deno project and effectively removing all typing checks and imports. 😕

Comment (gjsjohnmurray) [reply .vscode/settings.json]: No, SQLTools isn't using Deno AFAIK. I think it's better not to have settings in the project which relate to extensions we don't require. Instead I suggest you open the SQLTools workspace from its .code-workspace file, then go to your Extensions view, locate your Deno extension, click on its gearwheel and choose to disable it for the workspace. You might be asked to reload the workspace, but thereafter Deno shouldn't interfere with it even after you remove that extra line from /vscode/settings.json

**Review by gjsjohnmurray — APPROVED** [review]

**Comment by gjsjohnmurray** [comment]
Thanks for your contribution.

**Comment by robaho** [comment]
Thank you!

**Comment by gjsjohnmurray** [comment]
We just published new versions of these drivers. Thanks again for your contribution.

**Pull request merged**